feature-libs/asm/components/customer-emulation/customer-emulation.component.html

```diff
@@ -1,11 +1,58 @@
 <ng-container *ngIf="isCustomerEmulationSessionInProgress$ | async">
-  <input
-    formcontrolname="customer"
-    type="text"
-    disabled="true"
-    placeholder="{{ customer?.name }}, {{ customer?.uid }}"
-  />
-  <button (click)="logoutCustomer()">
+  <div class="customerInfo">
+    <label class="name">{{ customer?.name }}</label>
+    <label class="uid">{{ customer?.uid }}</label>
+  </div>
+  <label for="cartNumber"
+    >{{ 'asm.assignCart.cartNumber' | cxTranslate }}
+    <div
+      role="search"
+      aria-label="Assign a cart id to customer"
+      class="assignCart"
+      [class.active]="cartIdExists"
+      (click)="cartIdElement.focus()"
+    >
+      <input
+        autocomplete="off"
+        placeholder=""
+        #cartIdElement
+        formcontrolname="cartNumber"
+        [formControl]="cartId"
+        (keydown.enter)="assignCartToCustomer()"
+        aria-label="Enter cart id"
+      />
+      <button
+        class="reset"
+        aria-label="Reset"
+        [class.visible]="cartIdExists"
+        (click)="clearText()"
+      >
+        <cx-icon
+          class="cx-icon fas fa-times-circle"
+          ng-reflect-type="RESET"
```

Comment:
You should remove these - I can't find this being used anywhere else.
Also, removing them in my local, they render fine.